docs/docs/dev_docs/tutorials/token_portal/setup.md

```diff
@@ -0,0 +1,278 @@
+---
+title: Setup and Installation
+---
+
+In this step, we’re going to
+
+1. Install prerequisites
+2. Create a yarn project to house everything
+3. Create a nargo project for our Aztec contract
+4. Create a hardhat project for our Ethereum contract(s)
+5. Import all the Ethereum contracts we need
+6. Create a yarn project that will interact with our contracts on L1 and the sandbox
+
+We recommend going through this setup to fully understand where things live.
+
+However if you’d rather skip this part, our dev-rels repo contains the starter code here.
+
+# Prerequisites
+
+- [node v18+](https://github.com/tj/n)
+- [docker](https://docs.docker.com/)
+- [Aztec sandbox](https://docs.aztec.network/dev_docs/getting_started/sandbox) - you should have this running before starting the tutorial
+
+```bash
+/bin/bash -c "$(curl -fsSL 'https://sandbox.aztec.network')"
+```
+
+- Nargo
+
+```bash
+curl -L https://raw.githubusercontent.com/noir-lang/noirup/main/install | bash
+noirup -v #include_noir_version
+```
+
+# Create a yarn project
+
+Our root yarn project will house everything ✨
+
+```bash
+mkdir aztec-token-bridge && cd aztec-token-bridge && yarn init
+```
+
+In your `package.json` put this
```

Comment:
```suggestion
In your `package.json` add this
```
Just to be super clear, so people don't think `package.json` should be replaced with the following, only that it needs to be added to what's already there